client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler-starting-point/index.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+import { IntentScreen, StepContainer } from '@automattic/onboarding';
+import { useDispatch } from '@wordpress/data';
+import { useTranslate } from 'i18n-calypso';
+import startingPointImageUrl from 'calypso/assets/images/onboarding/starting-point.svg';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { preventWidows } from 'calypso/lib/formatting';
+import { ONBOARD_STORE } from '../../../../stores';
+import useStartingPoints from './use-starting-points';
+import type { Step } from '../../types';
+
+const PatternAssemblerStartingPointStep: Step = ( { navigation } ) => {
+	const { goBack, submit } = navigation;
+	const translate = useTranslate();
+	const headerText = translate( 'Nice job! Now it’s{{br}}{{/br}} time to get creative.', {
+		components: { br: <br /> },
+	} );
+	const subHeaderText = translate(
+		'Continue to the WordPress Site Editor or take a few moments to learn the basics.'
+	);
+	const startingPoints = useStartingPoints();
+	const { setStartingPoint } = useDispatch( ONBOARD_STORE );
+
+	const submitStartingPoint = ( startingPoint: string ) => {
+		const providedDependencies = { startingPoint };
+		recordTracksEvent( 'calypso_signup_pa_starting_point_select', {
+			starting_point: startingPoint,
+		} );
+		setStartingPoint( startingPoint );
+		submit?.( providedDependencies, startingPoint );
+	};
+
+	return (
+		<StepContainer
```

Comment:
I just noticed that we missed updating the document title on the steps:
- Design Picker
- Pattern Assembler
- Patter Assembler starting point
- Learn the basics

<img width="351" alt="Screen Shot 2565-11-10 at 11 59 11" src="https://user-images.githubusercontent.com/1881481/201005587-44821d28-4107-4f71-b234-be95f541349a.png">

We have to add it on top of the `<StepContainer>`:

```jsx
<>
  <DocumentHead title="The Step Document Title" />
  <StepContainer>
</>
```

Maybe we can fix them in another issue to discuss the title for each step.

Reply:
Oh yes, I remembered we have an issue for it